.github/workflows/mypy.yml

```diff
@@ -6,20 +6,26 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 5
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
I personally found that it costs 3 more CI times for this. I actually have doubt on whether it is necessary.

I feel like such strict running should only happen when we are ready to merge or the commit is already merged into the main branch.

Reply:
The idea case is that we have a trigger in PR template. when we click on this, it runs the full testing.

Reply:
Right, I feel we defs don't want really long CI time

Reply:
but I think it can be necessary when merging into the main branch. (my personal opinion) since vllm supports this.

Reply:
Yeah do you how to implement a github action that is required for merging, but not automatically activated in PR?

Reply:
And these three versions are running in parallel right?

Reply:
@ProKil yes, running in parallel but it costs the total organization's CI time (which is 2000 min/month for one free org). I actually run out of the monthly CI time cost for my own organization.

Reply:
> Yeah do you how to implement a github action that is required for merging, but not automatically activated in PR?

no, not yet. I am trying to figure this out recently. I will let you know once I found the solution.